libs/cdk/utils/directives/selectable-list/selection.service.ts

```diff
@@ -39,7 +41,7 @@ export class SelectionService<ElementType extends Element, ValueType = ElementTy
             map((value) => (this._isMultipleMode ? value : [value[0]])),
             map((coerced: ValueType[]) => coerced.filter(Boolean))
         );
-        this._normalizedValue$.pipe(takeUntil(this._destroy$)).subscribe((val) => (this._value = val));
+        this._normalizedValue$.pipe(takeUntilDestroyed(this._destroy$)).subscribe((val) => (this._value = val));
```

Comment:
It should be injection context, so no need for destroyRef here I think

Reply:
It is, but if we don't pass DestroyRef directly, `takeUntilDestroyed` will try to inject it again